HIP/hip-0000-mirror-node-account-id.md

```diff
@@ -0,0 +1,68 @@
+- hip: <HIP number (this is determined by the HIP editor)>
+- title: Message Account Id in Mirror Node API Response
+- author: scalemaildev
+- type: Standards Track
+- category: Mirror
+- status: Draft
```

Comment:
Once the HIP is ready for review by the editors, please change the status to `Review`